trunk/src/lsc/externaldata.py

```diff
@@ -263,14 +263,22 @@ def sdss_swarp(imglist,_telescope='spectral',_ra='',_dec='',output='', objname='
     import string
 
     if _telescope == 'spectral':
-        pixelscale = 0.30104  # 2 meter
```

Comment:
@estefaniapadilla can you fix the indentation that was removed when you modified this file? It should be 4 spaces